client/src/nv_ingest_client/util/milvus.py

```diff
@@ -912,10 +917,11 @@ def write_to_nvingest_collection(
     access_key: str = "minioadmin",
     secret_key: str = "minioadmin",
     bucket_name: str = "a-bucket",
-    threshold: int = 10,
+    threshold: int = 50,
```

Comment:
```suggestion
    threshold: int = 1000,
```

when testing w/ bo20, there are 812 chunks. with stream=True, total runtime goes from 114 to 76s

Seems like the threshold should be higher, which gives a nice perf lift for smaller ingest jobs